examples/amp-consent/amp-consent-iframe.amp.html

```diff
@@ -0,0 +1,317 @@
+    <script async custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"></script>
+    <script async custom-element="amp-form" src="https://cdn.ampproject.org/v0/amp-form-0.1.js"></script>
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script async custom-element="amp-consent" src="https://cdn.ampproject.org/v0/amp-consent-0.1.js"></script>
+    <script async custom-element="amp-iframe" src="https://cdn.ampproject.org/v0/amp-iframe-0.1.js"></script>
+  </head>
+  <body>
+
+    <div class="filterBar">
+      <form action="amp-consent-iframe.amp.html" method="get" target="_top">
+        <label for="filter">Vendor filter</label>
+        <select id="filter" name="type">
+          <option>[Select a vendor]</option>
+          <!-- This should be in alphabetical order. _ping_ should be the first -->
+          <option>_ping_</option>
+          <option>appconsent</option>
+          <option>didomi</option>
+          <option>sirdata</option>
+          <option>Marfeel</option>
+          <option>Ogury</option>
+          <option>opencmp</option>
+          <option>SourcePoint</option>
+          <option>Usercentrics</option>
+        </select>
+        <input type="submit" value="Go">
+        <button type="button" class="clearLocalStorage" onclick="localStorage.clear(); alert('localStorage Cleared!')">Clear localStorage</button>
+      </form>
+    </div>
+
+    <h1 id="top">AMP Consent Iframe</h1>
+
+    <amp-iframe
+      data-block-on-consent="_till_responded"
+      src="http://iframe.localhost:8000/examples/amp-consent/amp-consent-iframe.embed.html"
+      sandbox="allow-scripts allow-same-origin allow-popups"
+      allow="autoplay *; fullscreen *"
+      frameborder="0"
+      width="16"
+      height="9"
+      style="max-width:700px;"
+      layout="responsive">
+      <amp-img layout="fill" src="https://lh3.googleusercontent.com/pSECrJ82R7-AqeBCOEPGPM9iG9OEIQ_QXcbubWIOdkY=w300-h200-no-n" placeholder></amp-img>
+    </amp-iframe>
+
+    <div id="container">
+      Container for amp-consent tags.
+
+      <!-- This should be in alphabetical order. _ping_ should be the first -->
+
+      <!-- _ping_ example -->
```

Comment:
I think adding the fake CMP `_ping_` would be sufficient to test the feature. There's no need to add following `<amp-consent>` elements. 
In this case, you don't need to change to app.js either.